deploy/samconfig.toml

```diff
@@ -0,0 +1,11 @@
+version = 0.1
+[dev]
+[dev.deploy]
+[dev.deploy.parameters]
+stack_name = "backend-stelios"
```

Comment:
backend-main pls for all the references

Reply:
[Resolved](https://github.com/alphagov/di-ipv-cri-f2f-api/pull/9/commits/3982dce1affc187e059f5afa20fa756dc09fc904)